package.json

```diff
@@ -150,6 +150,33 @@
 						},
 						"yellow": {
 							"type": "string"
+						},
```

Comment:
Ok so one possibility for an alternate implementation is to not have any requirements for what the color names can be. Gives more flexibility, but might just make things more confusing for the user? Can't decide

Reply:
You mean just allow any key value in this map? Personally I would rather set my own names than have incorrect names and we can always ship et with default names of course that the user can change?

Reply:
Shall we close this one then?  That's gonna be a completely different implementation

Reply:
I want to try other solution first.

Reply:
I don't think this implementation without any fixed names are hard the problem is that without any predefined names we can't have default settings. There will be no way for them to edit the color or enable the setting without editing the json file.

Reply:
I've done some testing and as soon as you set `additionalProperties` to true you lose all settings gui for that. All the default values and existing settings are only accessible in the json file. 

As I see it we have two options:
1. Default names. The pros is that you can change color and enable them from the setting gui. The downside is that you can't select your own names. 
2. The user selects the names. Upside is that the user can choose their own names that match the color selection better. The downside is that you can't change enablement or color setting without editing the json file. The user can also spell the the color wrong in the csv files and we have no whitelist to reference against. 

I'm actually leaning towards the implementation in this pr. But we might want to reduce the number of colors to three or something. What do you think?